docs/source/index.rst

```diff
@@ -80,6 +80,7 @@ Some quick links:
    :hidden:
 
    operations/index.rst
+   operations/getting-started.rst
```

Comment:
@AlexandrePicosson In which category would you see it? My understanding was the section "Substra examples" and "Substrafl examples" were more deep dive into the use of the librairies. Should it be top-level?

Reply:
Maybe we could add a category targeted more at developers @adarbier what is your opinion on this ?

Reply:
Yes I think it would be great to have a "Contributing to Substra" section with: 
- High level guidelines for contributing
- How to locally run the substra stack (this doc basically)
- Maybe other sections? (build the doc, run the tests, submit a PR, etc.)

Reply:
For this PR I can create the category and move this page but I think the other points (including the front page of "Contributing to Substra") should be handled separately